src/test/regress/expected/rpt.out

```diff
@@ -875,7 +875,7 @@ explain (costs off, verbose) select * from t_hashdist left join t_replicate_vola
                      Output: t_replicate_volatile.a, t_replicate_volatile.b, t_replicate_volatile.c
          SubPlan 1  (slice2; segments: 3)
            ->  Materialize
-                 Output: random()
+                 Output: (random())
```

Comment:
plan was changed because materialize is done after explicit motion creation in make_subplan